src/main/java/ch/njol/skript/expressions/ExprDamage.java

```diff
@@ -39,47 +41,71 @@
 import ch.njol.util.Kleenean;
 import ch.njol.util.coll.CollectionUtils;
 
-/**
- * @author Peter Güttinger
- */
 @Name("Damage")
-@Description("How much damage is done in a damage event, possibly ignoring armour, criticals and/or enchantments. Can be changed (remember that in Skript '1' is one full heart, not half a heart).")
-@Examples({"increase the damage by 2"})
+@Description({
+	"How much damage is done in a damage event, possibly ignoring armour, criticals and/or enchantments.",
+	"Can be changed (remember that in Skript '1' is one full heart, not half a heart)."
+})
+@Examples({
+	"on damage of player:",
+		"\tdamage cause was lightning",
+		"\tvictim is wearing a leather helmet",
+		"\tsubtract 2.5 from the damage",
+		"\tmessage \"The smite was lessened by your leather helmet!\" to the victim"
+})
 @Since("1.3.5")
+@RequiredPlugins("Spigot 1.14+ for the item damage event.")
```

Comment:
Shorter, for better docs view
```suggestion
@RequiredPlugins("Spigot 1.14+ (item damage event)")
```

Reply:
I forgot about `Spigot` we usually use MC instead unless it's Paper, can you edit that as well?

Reply:
We use Spigot as we plan to support multiple platforms in the future. So we shouldn't be using MC but rather Paper or Spigot.

Reply:
Fair enough.